From daec6b1ac69ec2d2ce25473e3dc67b2485f4b058 Mon Sep 17 00:00:00 2001
From: Jason Lowe <jlowe@apache.org>
Date: Mon, 1 Dec 2014 21:51:05 +0000
Subject: [PATCH 2172/2748] MAPREDUCE-6172. TestDbClasses timeouts are too
 aggressive. Contributed by Varun Saxena (cherry
 picked from commit
 2b30fb1053e70c128b98013fb63cf9a095623be6)

(cherry picked from commit 2bdef5697bbed887861ea30c9d7891e98ea324cf)

Change-Id: I17b32db2eee679c3d39a9bdfa6dc5c90de90a84f
---
 .../hadoop/mapreduce/lib/db/TestDbClasses.java     |    8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/src/test/java/org/apache/hadoop/mapreduce/lib/db/TestDbClasses.java b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/src/test/java/org/apache/hadoop/mapreduce/lib/db/TestDbClasses.java
index 772552c..8169803 100644
--- a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/src/test/java/org/apache/hadoop/mapreduce/lib/db/TestDbClasses.java
+++ b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/src/test/java/org/apache/hadoop/mapreduce/lib/db/TestDbClasses.java
@@ -39,7 +39,7 @@
    * test splitters from DataDrivenDBInputFormat. For different data types may
    * be different splitter
    */
-  @Test(timeout = 1000)
+  @Test(timeout = 10000)
   public void testDataDrivenDBInputFormatSplitter() {
     DataDrivenDBInputFormat<NullDBWritable> format = new DataDrivenDBInputFormat<NullDBWritable>();
     testCommonSplitterTypes(format);
@@ -49,7 +49,7 @@ public void testDataDrivenDBInputFormatSplitter() {
     assertEquals(DateSplitter.class, format.getSplitter(Types.TIME).getClass());
   }
 
-  @Test(timeout = 1000)
+  @Test(timeout = 10000)
   public void testDataDrivenDBInputFormat() throws Exception {
     JobContext jobContext = mock(JobContext.class);
     Configuration configuration = new Configuration();
@@ -79,7 +79,7 @@ public void testDataDrivenDBInputFormat() throws Exception {
         configuration.get(DBConfiguration.INPUT_BOUNDING_QUERY));
   }
 
-  @Test(timeout = 1000)
+  @Test(timeout = 10000)
   public void testOracleDataDrivenDBInputFormat() throws Exception {
     OracleDataDrivenDBInputFormat<NullDBWritable> format = 
         new OracleDataDrivenDBInputFormatForTest();
@@ -96,7 +96,7 @@ public void testOracleDataDrivenDBInputFormat() throws Exception {
    * test generate sql script for OracleDBRecordReader.
    */
 
-  @Test(timeout = 2000)
+  @Test(timeout = 20000)
   public void testOracleDBRecordReader() throws Exception {
     DBInputSplit splitter = new DBInputSplit(1, 10);
     Configuration configuration = new Configuration();
-- 
1.7.9.5

